opentelemetry-exporters-newrelic-auto/src/test/java/com/newrelic/telemetry/opentelemetry/export/auto/NewRelicSpanExporterFactoryTest.java

```diff
@@ -17,19 +17,44 @@
 
   @Test
   void testFromConfig_HappyPath() {
-    String apiKeyKey = "newrelic.api.key";
+    String apiKeyKey = NewRelicConfiguration.NEW_RELIC_API_KEY;
     String apiKeyValue = "test-key";
-    String enableAuditLoggingKey = "newrelic.enable.audit.logging";
+    String enableAuditLoggingKey = NewRelicConfiguration.NEW_RELIC_ENABLE_AUDIT_LOGGING;
     String defaultServiceName = "(unknown service)";
-    String serviceNameKey = "newrelic.service.name";
+    String serviceNameKey = NewRelicConfiguration.NEW_RELIC_SERVICE_NAME;
     String serviceNameValue = "best service ever";
-    String uriOverrideKey = "newrelic.uri.override";
+    String uriOverrideKey = NewRelicConfiguration.NEW_RELIC_TRACE_URI_OVERRIDE;
     String uriOverrideValue = "http://test.domain.com";
 
     when(config.getString(apiKeyKey, "")).thenReturn(apiKeyValue);
     when(config.getBoolean(enableAuditLoggingKey, false)).thenReturn(true);
     when(config.getString(serviceNameKey, defaultServiceName)).thenReturn(serviceNameValue);
     when(config.getString(uriOverrideKey, "")).thenReturn(uriOverrideValue);
+    when(config.getString(NewRelicSpanExporterFactory.NEW_RELIC_URI_OVERRIDE, "")).thenReturn("");
+
+    NewRelicSpanExporterFactory newRelicSpanExporterFactory = new NewRelicSpanExporterFactory();
+    SpanExporter spanExporter = newRelicSpanExporterFactory.fromConfig(config);
+
+    assertNotNull(spanExporter);
+  }
+
+  @Test
+  void testFromConfig_OldUri() {
```

Comment:
Why "OldUri"?

Reply:
probably should be "oldProperty", rather than "oldUri". Will update.

Reply:
ah, gotcha.